src/tour.gleam

```diff
@@ -1,14 +1,18 @@
 import filepath
 import gleam/io
 import gleam/list
-import gleam/option.{type Option, None, Some}
+import gleam/option.{None, Some}
 import gleam/result
 import gleam/string
 import gleam/string_builder
 import htmb.{h, text}
 import simplifile
 import snag
-import tour/widgets
+import tour/types/lesson.{type Chapter, type Lesson, Chapter, Lesson}
+import tour/document
+import tour/page.{PageConfig, ScriptConfig}
+import tour/styles
+import tour/pages/everything
```

Comment:
Please revert splitting out these modules. We like to group code together that is used together, and having types modules is considered an anti-pattern.

Reply:
I had to move the types to a different file in order to avoir circular dependencies. I assume you want all the files' content (`document.gleam`, `page.gleam`, `everything.gleam`) merged into `tour.gleam`. Is that correct ? I agree with the grouping things that belong together in a single file, though I feel it souls make the whole codebase less readable. What do you think ?

Reply:
Yes please! All together.